README.md

```diff
@@ -5,6 +5,64 @@
 This is an ansible role to set up [vector](https://vector.dev).
 It translates the YAML configuration to TOML, so any configuration is possible.
 
-For available variables check out [defaults](roles/vector/defaults/main.yml)
-
 Currently only amd64, arch64, arch7 through deb and rpm packages are supported
+
+## Variables
+
+| Variable                                   | Required | Default                | Description
+|--------------------------------------------|----------|------------------------|------------
+| vector_template | yes | vector.toml.j2 | path of your vector.toml template
+| vector_config_file | yes | /etc/vector/vector.toml | system path of your vector.toml configuration
+| vector_nightly | no | false | use vector nightly build
+| add_vector_docker_group | no | false | add user vector to group docker
+| add_vector_journal_group | no | false | add user vector to group systemd-journal
+| add_vector_journal_group | no | false | add user vector to group systemd-journal
```

Comment:
small typeo: `add_vector_journal_group` mentioned twice

Reply:
I removed the duplicate in https://github.com/dzervas/ansible-vector/pull/17/commits/39c971980b48d37bbe44e7dd1167f760771cda52.